src/app/features/feedback-button/feedback-button.tsx

```diff
@@ -19,8 +19,8 @@ export function FeedbackButton() {
   const { theme } = useThemeSwitcher();
   return (
     <Button
-      background="ink.1"
-      _hover={{ background: 'ink.2' }}
+      background="oldInk.1"
```

Comment:
This works for me for now to label the `ink` as `oldInk`. We will need to work with design to figure out what this colour actually is and name it appropriately. 

In the meantime if we want we could come up with some random names like using a mnemonic key phrase or something. 

`oldInk.1` -> `achieve` 
`oldInk.2` -> `enjoy`